depthai_bridge/include/depthai_bridge/BridgePublisher.hpp

```diff
@@ -31,14 +27,23 @@ namespace rosOrigin = rclcpp;
     #include <boost/shared_ptr.hpp>
 
     #include "sensor_msgs/Image.h"
+#endif
+
+namespace dai {
+
+#ifdef IS_ROS2
+namespace StdMsgs = std_msgs::msg;
+namespace ImageMsgs = sensor_msgs::msg;
+using ImagePtr = ImageMsgs::Image::SharedPtr;
+namespace rosOrigin = rclcpp;
+#else
 namespace StdMsgs = std_msgs;
 namespace ImageMsgs = sensor_msgs;
 using ImagePtr = ImageMsgs::ImagePtr;
-namespace rosOrigin = ros;
+namespace rosOrigin = ::ros;
 #endif
```

Comment:
Probably better to go one layer deeper into `dai::ros` just so core doesn't clash with this by any chance (low chances, still nicely separated.
You'll still be able to access `rosOrigin` as is now, in `dai::ros`. 

`rosBridge` is okay as it needs to be `dai::rosBridge`